### PR TITLE
Neal Review

### PR DESCRIPTION
**Pull request by bmiller180787**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -55,15 +55,15 @@ header {
 
 .navbar {
     display: flex;
-    flex-wrap: wrap;
+/*    flex-wrap: wrap;*/
```
Comment (nealioa) [inline css/styles.css]: If you don't need it, delete it.

File: index.html [hunk 2]
```diff
@@ -133,5 +137,5 @@ <h4>Get In Touch</h4>
         </div>
     </footer>
 </body>
-
+
```
Comment (nealioa) [inline index.html]: Why are these spaces here? A blank line is ok, but spaces?

**Review by nealioa — APPROVED** [review]

File: css/styles.css (hunk 1 above)
Comment (bmiller180787) [reply css/styles.css]: I have amended the above

File: index.html (hunk 2 above)
Comment (bmiller180787) [reply index.html]: I have amended the above

**Review by nealioa — APPROVED** [review]

**Pull request merged**